internal/server/http.go

```diff
@@ -48,8 +50,17 @@ func newHttpServer(grpcServer *grpc.Server, ln net.Listener, opts *options) *htt
 	// gRPC server. This is used by the exec handler.
 	grpcAddr := opts.GRPCListener.Addr().String()
 
+	// Create grpc-gateway muxer
+	grpcMux := runtime.NewServeMux()
+	grpcOpts := []grpc.DialOption{grpc.WithTransportCredentials(insecure.NewCredentials())}
+	err := gen.RegisterWaypointHandlerFromEndpoint(opts.Context, grpcMux, "localhost:9701", grpcOpts)
+	if err != nil {
+		log.Error("Unable to register waypoint grpc gateway service")
```

Comment:
Hrm, I think we'd want to return this error too instead of only logging it?

Reply:
Resolved in `c9a3fd348844a628157b0e0167822bb14764024d`